ai21/models/chat/chat_message.py

```diff
@@ -0,0 +1,12 @@
+from dataclasses import dataclass
+
+from ai21.models import RoleType
```

Comment:
Maybe role type should be under `chat` and exported in `models`? This ways users won't need to guess where to import it from